chapter1.md

```diff
@@ -0,0 +1,36 @@
+# Chapter 1
+
+
+## What is web archiving?
+
+Web archiving is the process of recording web resources. Various elements of the website such as HTML, scripts, images, videos, etc. can be recorded to preserve as much of the original resource as possible. By creating dynamic archives that provide a user with the same experience they would have if they accessed the original site, web archives can give much more information than a static screenshot. 
+
+Web archives may be utilized by future researchers, historians and the general public. 
+
+## Why Autopilot?
+
+Autopilot is a tool that navigates a website similarly to how a human would, doing things like scrolling, clicking buttons, and playing videos. These actions executed through functions called "behaviors." Because Autopilot can go through the technically complicated aspects of a website, it is useful for recording high-fidelity websites.
+
+## Installing Autopilot
+Both [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) must be installed to run Autopilot. 
+
+To install Autopilot, use Terminal to clone the Webrecorder repository: 
+1. `git clone `
```

Comment:
We might want to highlight also building from source; not sure which is best. Is the docker impl. kept up to date? @ikreymer 

This isn't blocking for now though

Reply:
Yes, Docker implementation should be, it is how its deployed in production for Conifer, will double check that its working. Yeah, probably both should be mentioned.